pkgs/servers/monitoring/net-snmp/CVE-2015-5621.patch

```diff
@@ -0,0 +1,132 @@
+From 3714c5be3212d2af61545439eeb432e5d84a8d39 Mon Sep 17 00:00:00 2001
```

Comment:
Can we not `fetchPatch` this?

Reply:
I'll check.  In general I preferred, mostly for convenience, to make local copies of patches.

This isn't the "Nixpkgs Way", I'll see what can be fetchpatch'd here and elsewhere.